metadata-store-directory-common/src/main/java/org/apache/helix/msdcommon/constant/MetadataStoreRoutingConstants.java

```diff
@@ -20,6 +20,11 @@
  */
 
 public class MetadataStoreRoutingConstants {
+  public enum HttpRequestForwardingVerbs {
```

Comment:
If you want to do this, I think you could find HTTP verbs defined elsewhere already.